### PR TITLE
doc(release): prepare release notes for modules 2204

### PR DESCRIPTION
**Pull request by pkippes**
## Description

REF # MON-21511

Prepare release notes for:
* MAP 22.04.10

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x (staging)
- [x] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] 23.04.x (staging)
- [ ] Cloud (staging)
- [ ] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

### CODE REVIEW

**Review by smau-centreon — DISMISSED** [review]

**Review by smau-centreon — APPROVED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-commercial-extensions.md [hunk 1]
```diff
@@ -15,6 +15,15 @@ Pour faire des demandes d'évolutions ou reporter des bugs sur les extensions co
 
 ## Centreon MAP
 
+### 22.04.10
+
+Release date: `September 18, 2023`
+
+#### Bug fixes
+
+- [Packaging] Fixed a regression in centreon-map-server and centreon-map-engine packaging that caused a broken dependency situation with Java.
```
Comment (jdelpierre) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-commercial-extensions.md:24]: As there are 2 different chapters between Engine and Legacy, I will only mention the Engine package here.

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-commercial-extensions.md [hunk 2]
```diff
@@ -159,6 +168,15 @@ The new MAP extension is now available in a full web version with a new server,
 
 ## Centreon MAP Legacy
 
+### 22.04.10
+
+Release date: `September 18, 2023`
+
+#### Bug fixes
+
+- [Packaging] Fixed a regression in centreon-map-server and centreon-map-engine packaging that caused a broken dependency situation with Java.
```
Comment (jdelpierre) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-commercial-extensions.md:177]: As there are 2 different chapters between Engine and Legacy, I will only mention the Legacy package here.

File: versioned_docs/version-22.04/releases/centreon-commercial-extensions.md [hunk 3]
```diff
@@ -16,6 +16,15 @@ If you have feature requests or want to report a bug, please contact support.
 
 ## Centreon MAP
 
+### 22.04.10
+
+Release date: `September 18, 2023`
+
+#### Bug fixes
+
+- [Packaging] Fixed a regression in centreon-map-server and centreon-map-engine packaging that caused a broken dependency situation with Java.
```
Comment (jdelpierre) [inline versioned_docs/version-22.04/releases/centreon-commercial-extensions.md:25]: As there are 2 different chapters between Engine and Legacy, I will only mention the Engine package here.

File: versioned_docs/version-22.04/releases/centreon-commercial-extensions.md [hunk 4]
```diff
@@ -157,6 +166,15 @@ The new MAP extension is now available in a full web version with a new server,
 
 ## Centreon MAP Legacy
 
+### 22.04.10
+
+Release date: `September 18, 2023`
+
+#### Bug fixes
+
+- [Packaging] Fixed a regression in centreon-map-server and centreon-map-engine packaging that caused a broken dependency situation with Java.
```
Comment (jdelpierre) [inline versioned_docs/version-22.04/releases/centreon-commercial-extensions.md:175]: As there are 2 different chapters between Engine and Legacy, I will only mention the Legacy package here.

**Review by jdelpierre — APPROVED** [review]

**Review by smau-centreon — APPROVED** [review]

**Pull request merged**